[2023-06-08T18:20:50.653+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: wrapper_ex_dag.execute_function_task manual__2023-06-08T18:20:49.809090+00:00 [queued]>
[2023-06-08T18:20:50.658+0000] {taskinstance.py:1103} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: wrapper_ex_dag.execute_function_task manual__2023-06-08T18:20:49.809090+00:00 [queued]>
[2023-06-08T18:20:50.658+0000] {taskinstance.py:1308} INFO - Starting attempt 1 of 2
[2023-06-08T18:20:50.664+0000] {taskinstance.py:1327} INFO - Executing <Task(_PythonDecoratedOperator): execute_function_task> on 2023-06-08 18:20:49.809090+00:00
[2023-06-08T18:20:50.674+0000] {standard_task_runner.py:57} INFO - Started process 69 to run task
[2023-06-08T18:20:50.677+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'wrapper_ex_dag', 'execute_function_task', 'manual__2023-06-08T18:20:49.809090+00:00', '--job-id', '229', '--raw', '--subdir', 'DAGS_FOLDER/Python-Delta-Lake/custom_dags/WrapperDag.py', '--cfg-path', '/tmp/tmps9k5e9x6']
[2023-06-08T18:20:50.679+0000] {standard_task_runner.py:85} INFO - Job 229: Subtask execute_function_task
[2023-06-08T18:20:50.705+0000] {task_command.py:410} INFO - Running <TaskInstance: wrapper_ex_dag.execute_function_task manual__2023-06-08T18:20:49.809090+00:00 [running]> on host c712b1d8680f
[2023-06-08T18:20:50.745+0000] {taskinstance.py:1545} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='phomsophaN@saccounty.net' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='wrapper_ex_dag' AIRFLOW_CTX_TASK_ID='execute_function_task' AIRFLOW_CTX_EXECUTION_DATE='2023-06-08T18:20:49.809090+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2023-06-08T18:20:49.809090+00:00'
[2023-06-08T18:20:50.745+0000] {logging_mixin.py:149} INFO - CSV Delta Class Initialted
[2023-06-08T18:20:50.745+0000] {CsvDagWrapper.py:23} INFO - CSV Delta Class Initialted
[2023-06-08T18:20:50.745+0000] {logging_mixin.py:149} INFO - Environment successfully parsed! Environment is Local
[2023-06-08T18:20:50.745+0000] {CsvDagWrapper.py:29} INFO - Environment successfully parsed! Environment is Local
[2023-06-08T18:20:50.745+0000] {logging_mixin.py:149} INFO - Table/s successfully parsed! Table:  CASE
[2023-06-08T18:20:50.745+0000] {CsvDagWrapper.py:34} INFO - Table/s successfully parsed! Table:  CASE
[2023-06-08T18:20:50.746+0000] {CsvDagWrapper.py:36} INFO - Works so far....
[2023-06-08T18:20:52.904+0000] {taskinstance.py:1824} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/decorators/base.py", line 220, in execute
    return_value = super().execute(context)
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 181, in execute
    return_value = self.execute_callable()
  File "/home/airflow/.local/lib/python3.10/site-packages/airflow/operators/python.py", line 198, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/opt/airflow/dags/Python-Delta-Lake/custom_dags/WrapperDag.py", line 46, in execute_function_task
    task.RunCsvToDelta("Local","CASE",False)
  File "/opt/airflow/dags/Python-Delta-Lake/Csv2Delta/CsvDagWrapper.py", line 37, in RunCsvToDelta
    spark = SparkApp.GetApp(f"Csv To Delta Main {environment_location_suffix}")
  File "/opt/airflow/dags/Python-Delta-Lake/Csv2Delta/SparkApp.py", line 12, in GetApp
    spark = CD(spark_builder).enableHiveSupport().getOrCreate()
  File "/home/airflow/.local/lib/python3.10/site-packages/delta/pip_utils.py", line 66, in configure_spark_with_delta_pip
    raise TypeError(msg)
TypeError: 
This function must be called with a SparkSession builder as the argument.
The argument found is of type <class 'pyspark.sql.session.SparkSession'>.
See the online documentation for the correct usage of this function.
        
[2023-06-08T18:20:52.910+0000] {taskinstance.py:1345} INFO - Marking task as UP_FOR_RETRY. dag_id=wrapper_ex_dag, task_id=execute_function_task, execution_date=20230608T182049, start_date=20230608T182050, end_date=20230608T182052
[2023-06-08T18:20:52.918+0000] {standard_task_runner.py:104} ERROR - Failed to execute job 229 for task execute_function_task (
This function must be called with a SparkSession builder as the argument.
The argument found is of type <class 'pyspark.sql.session.SparkSession'>.
See the online documentation for the correct usage of this function.
        ; 69)
[2023-06-08T18:20:52.935+0000] {local_task_job_runner.py:225} INFO - Task exited with return code 1
[2023-06-08T18:20:52.944+0000] {taskinstance.py:2651} INFO - 0 downstream tasks scheduled from follow-on schedule check
